SDWebImage/Core/SDWebImageDownloader.m

```diff
@@ -353,8 +353,13 @@ + (SDWebImageOptions)imageOptionsFromDownloaderOptions:(SDWebImageDownloaderOpti
     
     // Operation Class
     Class operationClass = self.config.operationClass;
-    if (operationClass && [operationClass isSubclassOfClass:[NSOperation class]] && [operationClass conformsToProtocol:@protocol(SDWebImageDownloaderOperation)]) {
-        // Custom operation class
+    if (operationClass) {
+        if ([operationClass isSubclassOfClass:[NSOperation class]] && [operationClass respondsToSelector:@selector(initWithRequest:inSession:options:context:)]) {
+            // Custom operation class
+        } else {
+            // fallback
```

Comment:
This fallback here because 5.14.0 we introduce new API, I think this need some time to adopt